app/src/main/java/de/danoeh/antennapod/fragment/actions/FeedMultiSelectActionHandler.java

```diff
@@ -0,0 +1,189 @@
+        preferenceListDialog.setOnPreferenceChangedListener(which -> {
+            FeedPreferences.AutoDeleteAction autoDeleteAction = null;
+            switch (items[which]) {
+                case "global":
+                    autoDeleteAction = FeedPreferences.AutoDeleteAction.GLOBAL;
+                    break;
+                case "always":
+                    autoDeleteAction = FeedPreferences.AutoDeleteAction.YES;
+                    break;
+                case "never":
+                    autoDeleteAction = FeedPreferences.AutoDeleteAction.NO;
+                    break;
+            }
+            FeedPreferences.AutoDeleteAction finalAutoDeleteAction = autoDeleteAction;
+            saveFeedPreferences(feedPreferences -> {
+                feedPreferences.setAutoDeleteAction(finalAutoDeleteAction);
+            });
+        });
+    }
+
+    private void keepUpdatedChecked() {
+        PreferenceSwitchDialog preferenceSwitchDialog = new PreferenceSwitchDialog(activity,activity.getString(R.string.kept_updated), activity.getString(R.string.keep_updated_summary));
+        preferenceSwitchDialog.setOnPreferenceChangedListener(new PreferenceSwitchDialog.OnPreferenceChangedListener() {
+            @Override
+            public void preferenceChanged(boolean enabled) {
+                saveFeedPreferences(feedPreferences -> feedPreferences.setKeepUpdated(enabled));
+            }
+        });
+        preferenceSwitchDialog.openDialog();
+    }
+
+
+
+    private void deleteChecked() {
+        RemoveFeedDialog.show(activity, selectedItems, null);
+    }
+
+    private void showMessage(@PluralsRes int msgId, int numItems) {
+        activity.showSnackbarAbovePlayer(activity.getResources()
+                .getQuantityString(msgId, numItems, numItems), Snackbar.LENGTH_LONG);
+    }
+
+    private void showMessageMore(@PluralsRes int msgId, int countNoMedia, int countHasMedia) {
+        activity.showSnackbarAbovePlayer(activity.getResources()
+                .getQuantityString(msgId,
+                        (countHasMedia + countNoMedia),
+                        (countHasMedia + countNoMedia), countHasMedia),
+                Snackbar.LENGTH_LONG);
+    }
+
+    private long[] getSelectedIds() {
+        long[] checkedIds = new long[selectedItems.size()];
+        for (int i = 0; i < selectedItems.size(); ++i) {
+            checkedIds[i] = selectedItems.get(i).getId();
+        }
+        return checkedIds;
+    }
+    interface OnSetFeedPreferenceListener {
+        void onSetFeedPreferenceListener(FeedPreferences feedPreferences);
+    }
+    private void saveFeedPreferences(OnSetFeedPreferenceListener onSetFeedPreferenceListener) {
+        for (Feed feed : selectedItems) {
+            onSetFeedPreferenceListener.onSetFeedPreferenceListener(feed.getPreferences());
+            DBWriter.setFeedPreferences(feed.getPreferences());
+        }
+    }
+
+    private class ApplyToEpisodesDialog extends ConfirmationDialog {
```

Comment:
That dialog is gone on the `master` branch. Please merge the branch and remove the confirmation dialog.